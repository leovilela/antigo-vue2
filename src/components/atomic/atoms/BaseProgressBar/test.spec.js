import { mount } from "@vue/test-utils";
import BaseProgress from "./index";

describe("Component BaseProgress <base-progress />", () => {
  const build = (props) => {
    const wrapper = mount(BaseProgress, {
      propsData: { ...props },
    });

    return {
      wrapper,
    };
  };

  const type = "creditLimit";
  const { wrapper } = build();

  beforeEach(() => wrapper.destroy());

  test("mount the component", () => {
    expect(wrapper.isVueInstance()).toBe(true);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
