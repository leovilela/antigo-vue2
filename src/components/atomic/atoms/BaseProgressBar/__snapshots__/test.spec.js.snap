// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component BaseProgress <base-progress /> mount the component 1`] = `
<div class="base-progress-bar">
  <!----> <span class="base-progress-bar__progress" style="width: 0%;"></span>
  <!---->
  <!---->
</div>
`;
